container/BUILD

```diff
@@ -102,6 +102,8 @@ TEST_TARGETS = [
     "build_with_tag",
     "with_passwd",
     "with_group",
+    "with_empty_files",
+    "with_empty_dirs",
```

Comment:
Can you also add tests to the `image_test.py` in this directory that verify these exist in the resulting image?

Reply:
Hi @mattmoor, I am not sure I understand your request. I added a test that verifies that the image contains exactly expected files/dirs, that they have expected mode, and that files are empty. What am I missing?

Reply:
@pawelz that's perfect, thanks.  Sorry I missed that the first time!